### PR TITLE
change the wait

### PR DESCRIPTION
**Pull request by wild-endeavor**
# TL;DR
Try a different syntax for the publish wait.

### CODE REVIEW

**Review by pingsutw — APPROVED** [review]

File: .github/workflows/pythonpublish.yml [hunk 1]
```diff
@@ -57,10 +57,11 @@ jobs:
             echo "No tagged version found, exiting"
             exit 1
           fi
-          LINK="https://pypi.org/project/flytekitplugins-pod/${VERSION}"
+          LINK="https://pypi.org/project/flytekitplugins-pod/${VERSION}/"
           for i in {1..60}; do
-            if curl -L -I -s -f ${LINK} >/dev/null; then
-              echo "Found pypi"
+            result=$(curl -L -I -s -f ${LINK})
```
Comment (eapolinario) [inline .github/workflows/pythonpublish.yml:62]: What's the difference between these two?

**Review by eapolinario — COMMENTED** [review]
I don't think this fixes the underlying issue? We should have a 5 minute sleep before trying this as a last resort.

File: .github/workflows/pythonpublish.yml (hunk 1 above)
Comment (wild-endeavor) [reply .github/workflows/pythonpublish.yml:62]: nothing - but it prints the link next.  and explicitly check $? - not sure if that's what the if was doing before.

Comment (eapolinario) [reply .github/workflows/pythonpublish.yml:63]: This was doing this already, but this code you wrote is more clear to the uninitiated. I like it.

**Review by eapolinario — APPROVED** [review]

**Pull request merged**